config/dev.webpack.config.js

```diff
@@ -5,18 +5,18 @@ const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
     debug: true,
-    ...(process.env.BETA && { deployment: 'beta/apps' }),
+    ...(process.env.BETA && { deployment: 'preview/apps' }),
```

Comment:
@johnsonm325 only the actual browser routing will change. None if the CDN asset structure will be switching from beta to preview. This change is the reason why you can't run the UI locally.